src/commonMain/kotlin/komem/litmus/statistics/Dev.kt

```diff
@@ -0,0 +1,30 @@
+package komem.litmus.statistics
```

Comment:
What does `Dev` stands for in the name of this file?

Reply:
deleted this file